MediawikiHubot/MediawikiHubot.php

```diff
@@ -141,6 +141,26 @@ function hubot_incoming_new_user_account($user, $byEmail)
 }
 
 /**
+ * Called after a user account or IP is blocked
+ * @see https://www.mediawiki.org/wiki/Manual:Hooks/BlockIpComplete
+ */
+function hubot_incoming_user_blocked(Block $block, $user)
+{
+  global $wgWikiUrl, $wgWikiUrlEnding, $wgWikiUrlEndingBlockList;
+
+  $message = sprintf(
+    "%s has blocked %s %s Block expiration: %s. %s",
```

Comment:
I think block expiration is overkill here.

Reply:
Agreed, removed it